tap_zoom/sync.py

```diff
@@ -43,6 +43,16 @@ def sync_recordings(client,
                     key_bag,
                     parent_endpoint=None,
                     records=[]):
+    persist = endpoint.get('persist', True)
+    if persist:
```

Comment:
I think this was a holdover from the public tap, will remove

Reply:
This is how the public zoom tap was set up, although we don't utilize it for the recordings stream. There are a few endpoints that have `persist: False` set, so I'm going to keep this code in case we ever need to enable those endpoints.

<img width="616" alt="Screen Shot 2023-06-26 at 4 20 34 PM" src="https://github.com/Pathlight/tap-zoom/assets/1455838/15605ee9-de53-4571-ae0c-b8fce8ce5067">

Reply:
Ah nvm, I see what you're saying. Will remove for this `sync_recordings` method